docs/source/contributing/developing.adoc

```diff
@@ -131,10 +131,9 @@ If you seeing Glide errors during `make vendor`, you can clear the Glide cache v
 == Building Minishift
 
 [[build-minishift-binary]]
-=== Building the Minishift binary
+=== Building the Minishift Binary
```

Comment:
Right @LalatenduMohanty , the rules for title-case capitalization are not to capitalize articles and also some prepositions such as "in", "on". 

@gbraad I don't know what you are referring to, I see the titles on the master branch and they're capitalized correctly. Where did you spot this?